src/styles/base/_05-focus.scss

```diff
@@ -0,0 +1,34 @@
+// Focus state mixin
+@mixin focus-outline($darkBackground: false) {
+    outline: 2px solid var(--ui-focus);
+    outline-offset: 2px;
+
+    @if $darkBackground {
+        outline-color: var(--ui-white);
+    }
+}
+
+// Apply :focus behavior to focusable elements only (For IE 11)
+input,
```

Comment:
Looking through your comment here:  https://github.com/NYPL-Simplified/circulation-patron-web/pull/119#issuecomment-680907809

Does this mean that anybody that uses DS is getting these styles, because all CSS is automatically pulled in?  

Even though I think that visible focus is a desirable thing, I do want to give people the option to use their own focus styles if they want to.  To that end, it would make more sense to put these under a classname rather than on the elements themselves.

Reply:
That's what I'm thinking too. I think I'm going to use `.nypl-ds` as a namespace. I'll make those updates and get this in.